.github/workflows/release.yml

```diff
@@ -174,10 +174,13 @@ jobs:
           pattern: artifacts-*
           path: target/distrib/
           merge-multiple: true
+
       - name: Compile installer/uninstaller
         run: |
           TARGET_TRIPLE="$(rustc -Vv | awk '/^host/ { print $2 }')"
-          AMBER="$PWD/target/distrib/amber-${TARGET_TRIPLE}/amber"
+          AMBER_PACKAGE="$PWD/target/distrib/amber-${TARGET_TRIPLE}"
```

Comment:
I don't feel qualified to review this file, as I don't know a lot about CI servers; but the rest looks ok.